object-store/1.2/modules/ROOT/pages/object-store-connector-troubleshooting.adoc

```diff
@@ -0,0 +1,63 @@
+= Troubleshooting Object Store Connector - Mule 4
+
+To troubleshoot Anypoint Connector for Object Store (Object Store Connector), become familiar with the information about enabling verbose logging and interpreting commonly thrown messages.
+
+== Enable Verbose Logging
+
+To get a better understanding of why an application's interaction with Object Store Connector, temporarily enable verbose logging for the connector. +
+
+Remember to always disable the enhanced verbosity after troubleshooting, because it affects your Mule application performance.
+
+To enable verbose logging in the configuration file:
+
+. Access Anypoint Studio and navigate to the *Package Explorer* view.
+. Open your application's project name.
+. Open the `src/main/resources` path folder.
+. Open the `log4j2.xml` file inside the folder.
+. Add an `<AsyncLogger>` tag inside the `<Loggers>` tag:
++
+[source,xml,linenums]
+----
+			<Loggers>
+				...
+				<AsyncLogger name="org.mule.extension.objectstore" level="DEBUG"/>
+				...
+			</Loggers>
+----
+[start=6]
+. Save your application changes.
+. Click the project name in *Package Explorer* and then click *Run* > *Run As* > *Mule Application*.
+
+== Understand Common Throws
+
+Here is a list of common throw messages and how to interpret them:
+
+* OS:KEY_ALREADY_EXISTS
+
+ An object is trying to be stored, but the object store already has a value for that key.
```

Comment:
Can you rephrase this? An object is not trying to be stored. The system is trying to store an object.